underfs/s3/src/test/java/alluxio/underfs/s3/S3UnderFileSystemTest.java

```diff
@@ -14,22 +14,27 @@
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
-import org.powermock.api.mockito.PowerMockito;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 import org.powermock.reflect.Whitebox;
 
 /**
  * Tests for the private helper methods in {@link S3UnderFileSystem} that do not require an S3
  * backend.
  */
-public class S3UnderFileSystemTest {
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(S3UnderFileSystem.class)
+public final class S3UnderFileSystemTest {
```

Comment:
there is right now no harness to mock s3 (or GCS, OSS) client and test this. It requires a lot of new code. Not sure if I should make it in this PR as this is 1.1.1 blocking.
On the other hand, current tests coverage of ufs module is pretty low. I will create a ticket on this.